app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::CommentsController < ApplicationController
+  include Paginable
+  skip_before_action :authorize!, only: [:index]
+  before_action :load_article
+ 
+  def index
+    paginated = paginate(@article.comments)
+    render_collection(paginated)
+  end
+
+  def create
+    comment = @article.comments.create(comment_params.merge(user: current_user))
+
+    if comment.save
+      render json: serializer.new(comment), status: :created
+    else
+      render json: comment.errors, status: :unprocessable_entity
```

Comment:
Would like to update this error handling if time allows.  Should match similar code in ArticleController.